docs/using_pyodide_from_javascript.md

```diff
@@ -64,4 +64,58 @@ pyodide.loadPackage('matplotlib').then(() => {
 
 ## Complete example
 
-TODO
+Grab the main pyodide.js and supporting files referenced in the demo project (https://alpha.iodide.io/notebooks/300/)
+
+```bash
+mkdir pyodide_local ; cd pyodide_local
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.js
+curl -O https://alpha.iodide.app/pyodide-0.10.0/packages.json
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.js
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.wasm
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.data
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.data.js
+```
+
+Create and save a test `index.html` page (in the pyodide_local directory) with the following contents:
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <script type="text/javascript">
+        window.languagePluginUrl = 'http://localhost:8000/'; // set the pyodide support files (packages.json, pyodide.asm.data etc) url
+    </script>
+    <script src="pyodide.js"></script>
+</head>
+<body>
+  Pyodide test page <br>
+  Open your browser console to see pyodide output
+  <script type="text/javascript">
+        languagePluginLoader.then(function () {
+            console.log(pyodide.runPython('import sys\nsys.version'));
+            console.log(pyodide.runPython('help("modules")'));
+        });
+  </script>
+</body>
+```
+
+Unfortunately, because browsers require WebAssembly files to have mimetype of application/wasm we're unable to serve our files using Python's built-in SimpleHTTPServer module.
+
+Let's wrap Python's Simple HTTP Server and provide the appropiate mimetype for WebAssembly files into a `pyodide_server.py` file (in the pyodide_local directory):
+```python
+import BaseHTTPServer, SimpleHTTPServer
+
+SimpleHTTPServer.SimpleHTTPRequestHandler.extensions_map['.wasm'] = 'application/wasm'
```

Comment:
I don't think monkey-patching this is the supported way to do this from outside.  Better to use the documented API in `mimetypes`:

```
import mimetypes
mimetypes.init()
mimetypes.type_map['.wasm'] = 'application/wasm'
```

Reply:
Hi, I see where you're coming from. This seems to be python version dependent for some reason.

On my MacOSX 10.13.6, with default python 2.7, doing it properly (via mimetypes) doesn't seem to work.

Reply:
Ah.  We probably shouldn't be writing any examples with Python 2 in 2019 -- it sunsets at the beginning of 2020.

Let's update to use the `mimetypes` API and add a note in the docs to use Python 3.